test/README.md

```diff
@@ -13,6 +13,15 @@ environment.
 1. Setup a development environment as describe in
    [developer quick start guide](../docs/devel-quick-start.md)
 
+1. Install `libvirt`
+
+   ```
+   sudo dnf install @virtualization
+   ```
+
+   For more info see
+   [Virtualization – Getting Started](https://docs.fedoraproject.org/en-US/quick-docs/virtualization-getting-started/)
```

Comment:
We need to sync https://github.com/RamenDR/ramen/blob/main/docs/user-quick-start.md which show different instructions and use different link name.

Reply:
Added #1789 for the docs work.